inst/qml/designOfExperiments.qml

```diff
@@ -259,23 +298,61 @@ Form
 					max:						50
 				}
 			}
-
 		}
 	}
 
 	Section
 	{
 		visible: 								[1].includes(design.currentIndex)
 		title: 									qsTr("Screening Design Options")
+		columns:								1
 
-	}
+		RadioButtonGroup
+		{
+			name: 								"screeningType"
+			title:								qsTr("Type of Screening Design")
 
+			RadioButton
+			{
+				name:							"screeningTypePlackettBurman"
+				label:							qsTr("Plackett-Burman design")
+			}
+
+			RadioButton
+			{
+				name:							"screeningTypeTaguchi"
+				label:							qsTr("Taguchi design")
+			}
+		}
+	}
 
 	Section
 	{
 		visible: 								[2].includes(design.currentIndex)
 		title: 									qsTr("Response Surface Design Options")
 		
+		GroupBox
+		{
+			title: 							qsTr("Additional options")
+
+			IntegerField
```

Comment:
You don't neccesarily have to make these fields again, but you maybe recycle the existing options with the same names?